qa/automation/CareerPortal.Tests.Scenarios/Artemova/1.2 SendCV.feature

```diff
@@ -0,0 +1,138 @@
+Feature: 1.2 Send CV Form
+
+Background: 
+  Given user is on the <page> with position description
+  And the 'Send CV' form is available
+  And the 'Send CV' form doesn't contain any data
+	
+Examples:
+  | page                                                                  |
+  | https://career.quantori.com/positions/senior-qa-automation-c-engineer |
+
+Scenario Outline: 1.2.1 User enters all valid data
+  When user enters valid <Name> into the 'Name' field
+  And user enters valid <Email> into the 'Email' field
+  And user clicks 'Attach CV' button and loads a valid <File>

```

Comment:
Step should be separated into two steps
1. And user clicks 'Attach CV' button
2. And user loads a valid <File>